website/docs/tutorials/06-deploying/02-migrations/deploy_contract.md

```diff
@@ -0,0 +1,37 @@
+# `deploy_contract`
+
+```python
+def deploy_contract(
+    self,
+    contract_path: str,
+    constructor_args: Optional[List[int]] = None,
```

Comment:
It does not support data transformer?

Reply:
Nope. I didn't want to add support for in the first PR.